pyproject.toml

```diff
@@ -7,11 +7,18 @@ license = "GNU Affero General Public License v3.0"
 readme = "README.md"
 
 [tool.poetry.dependencies]
-python = "^3.12"
+python = "^3.10"
 django = "^5.1"
 django-environ = "^0.11.2"
 psycopg2-binary = "^2.9.9"
 flake8 = "^7.1.1"
+qdrant-client = "^1.11.1"
+langchain = "^0.2.15"
+langchain-community = "^0.2.14"
+langchain-openai = "^0.1.23"
+sentence-transformers = "3.0.1"
```

Comment:
After discussion with @ranjan-stha, We are planning to separate this out as standalone using fastapi server similar to ollama

Reply:
The embedding module is now standalone served from fastapi server.